platform/linux-dpdk/include/odp/api/feature.h

```diff
@@ -0,0 +1 @@
+../../../../linux-generic/include/odp/api/feature.h
```

Comment:
Agreed. @brbrooks mentioned that he can help in this area. We can discuss this during tomorrow's call. Perhaps open an Issue to track this? In any event, this would be a good starting point for work prior to any directory reorganization.